src/components/timer-ticking.ts

```diff
@@ -84,12 +84,11 @@ export default class TimerTicking extends EventManager {
     update(deltaTime) {
         if (this.startMyTimer && !this.isStoneDropped) {
             this.timer += deltaTime * 0.008;
-
             // Calculate the new width percentage for the timer
             const timerDepletion = Math.max(0, 100 - this.timer);
             this.timerFullContainer.style.width = `${timerDepletion}%`;
 
-            if (timerDepletion < 5 && !this.isMyTimerOver) {
+            if (timerDepletion < 10 && !this.isMyTimerOver) {
```

Comment:
+1

Reply:
updated test case

Reply:
@ashwinnair-chimple i think you need to use the update function instead of putting the whole logic in test file src/components/timerHtml/timerHtml.spec.ts

```
 if (expectedTimerDepletion < 10 && !timerTicking.isMyTimerOver) {
        expect(timerTicking.audioPlayer.playAudio).toHaveBeenCalledWith(AUDIO_TIMEOUT);
        expect(timerTicking.playLevelEndAudioOnce).toBe(false);
    }

    // Check if the timer is over and callback is called when timerDepletion <= 0
    if (expectedTimerDepletion <= 0) {
        expect(timerTicking.isMyTimerOver).toBe(true);
        expect(mockCallback).toHaveBeenCalledWith(true);
    }
````

Reply:
timerHtml.spec.ts file holds the logic of time-ticking.ts only as it is a baseHTML extended file I included the test case there along with timerHtml.spec.ts, @woodsjohn-codev  you can ignore this now and just review the changes made, As we have just made changes to play audio from 10% to 0% which was previously 5% to 0%.